PROTO_tests/tests/numeric_test.py

```diff
@@ -0,0 +1,362 @@
+import arkouda as ak
+import pytest
+import numpy as np
+
+from arkouda.dtypes import npstr
+
+NUMERIC_TYPES = [ak.int64, ak.float64, ak.bool, ak.uint64]
+CAST_TYPES = [ak.int64, ak.float64, ak.bool, ak.uint64, npstr]
+
+VALID_CAST = [
```

Comment:
i don't love the name `VALID_CAST` since these aren't the only valid casts, I think `roundtripable` was more descriptive because it's basically saying that these type combos can be casts both way (i.e. `int -> float` or `float -> int` both work)

Reply:
Renamed to `ROUNDTRIP_CAST`